src/pre_defined_classes/programs_with_changing_behaviour/hard/q2/Question2.java

```diff
@@ -0,0 +1,19 @@
+package pre_defined_classes.programs_with_changing_behaviour.hard.q2;
+import java.util.Random;
+
+/*
+Make a program that will generate a random <strong>8 digit</strong> number and encrypt it by dividing it by a second
+random <strong>1-digit</strong> number. At the end of the program, print out these numbers.
+Sample Output: 68056441 has been encrypted using 5 into 13611288
+ */
+
+public class Question2 {
```

Comment:
We should name these classes what they are called in the question (unless this is the actual name?)

Reply:
I changed the class names to what they are on the document